table/server/common/src/test/java/alluxio/table/common/udb/UdbConfigurationTest.java

```diff
@@ -50,9 +50,18 @@ public void multipleUfsMountOptions() {
   public void mountOptions() {
     testMountOptions("SCHEME" + randomString(), true);
     testMountOptions("SCHEME" + randomString(), false);
+    testMountOptions(".*", "SCHEME" + randomString(), true);
+    testMountOptions(".*", "SCHEME" + randomString(), false);
+    testMountOptions("SCHE.E.*", "SCHEME" + randomString(), true);
+    testMountOptions("SCHE.E.*", "SCHEME" + randomString(), false);
   }
 
   private void testMountOptions(String schemeAuthority, boolean specifyTrailingSlash) {
+    testMountOptions(schemeAuthority, schemeAuthority, specifyTrailingSlash);
+  }
+
+  private void testMountOptions(String schemeAuthority, String userSchemeAuthority,
+      boolean specifyTrailingSlash) {
```

Comment:
Since `schemeAuthority` can be a regex expression, it's better to reflect that in the variable name. e.g. `templateSchemeAuthority` vs `concreteSchemeAuthority`. Also this in the`UdbConfiguration` constructor.

Reply:
Done